modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -39,3 +44,43 @@ test('LPS-204541 check export/import menu visibility', async ({
 		usersAndOrganizationsPage.manageCustomFieldsOptionsMenuItem
 	).toHaveCount(0);
 });
+
+test('LPD-15224 check escape of memberships account name', async ({
+	apiHelpers,
+	editUserPage,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	await page.goto('/');
+
+	const account = await apiHelpers.headlessAdminUser.postAccount({
+		name: '<img src="x" onError="alert(document.location)">',
+	});
+
+	await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account.id,
+		['test@liferay.com']
+	);
+
+	try {
+		await usersAndOrganizationsPage.goToUsers();
+
+		await (
+			await usersAndOrganizationsPage.usersTableRowLink('test')
+		).click();
+		await editUserPage.membershipsLink.click();
+
+		await expect(
```

Comment:
how do you guarantee that this is not running the js malicious code?

Reply:
because I can find the malicious account name in page as text and it's not "converted" in the relative html tags